packages/cli/src/cli/index.ts

```diff
@@ -32,13 +32,75 @@ yargs(hideBin(process.argv))
           type: "string",
           choices: ["nextjs", "gatsby"],
           demandOption: true,
+        })
+        .option("appName", {
+          describe: '"package.json" compatible name for the project.',
+          type: "string",
+          demandOption: false,
+        })
+        .option("noInstall", {
+          describe: "Do not install any dependencies.",
+          type: "boolean",
+          default: false,
+          demandOption: false,
+        })
+        .option("use-npm", {
+          describe: "Use NPM package manager for installing dependencies.",
+          type: "boolean",
+          default: false,
+          demandOption: false,
+        })
+        .option("use-pnpm", {
+          describe: "Use PNPM package manager for installing dependencies.",
+          type: "boolean",
+          default: false,
+          demandOption: false,
+        })
+        .option("use-yarn", {
+          describe: "Use Yarn package manager for installing dependencies.",
+          type: "boolean",
+          default: true,
```

Comment:
Yarn does not come installed by default on Node distributions, I imagine NPM should be what we use as default.